### PR TITLE
14.0 mig purchase warn message

### PR DESCRIPTION
**Pull request by mostafaala2**
Migration.

### CODE REVIEW

**Comment by kittiu** [comment]
Thanks for your contribution. Just some contribution practice to add.
* Please change the title of this PR to `[14.0][MIG] purchase_warn_message`, this help searching PRs easier
* You don't need to migrate index.html and README.rst, you can even remove them, as they will be created by files in readme on merge.

File: purchase_warn_message/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module add a warning popup on purchase orders to ensure warning is
+populated no only when partner is changed.
```
Comment (kittiu) [inline purchase_warn_message/readme/DESCRIPTION.rst:2]:
```suggestion
populated not only when partner is changed.
```

Comment (kittiu) [reply purchase_warn_message/readme/DESCRIPTION.rst:2]: In addition, can we add more description to ease understanding? Myself, never use the warning, and don't understand until I look into the code. May be something like this?

```
When "Warnings" is enable for purchase order, each partner can be setup for a warning message.

This module add a persistent warning popup on purchase orders to ensure warning is
populated not only when partner is changed.
```

**Comment by felipemotter** [comment]
Can this PR be merged?

**Review by felipemotter — APPROVED** [review]

**Pull request closed without merging**